spec/game_spec.rb

```diff
@@ -39,15 +39,42 @@
   end
 
   it 'keeps track of player turns' do
-    game = game_setup([1, 2])
+    game = game_setup(FakeUi.new([0, 1]))
 
     2.times { game.play_turn }
 
     expect(board.player_at(1)).not_to eq(board.player_at(0))
   end
 
+  it 'gets Ui to display the board' do
+    ui = FakeUi.new([0])
+    game = game_setup(ui)
+
+    game.play_turn
+
+    expect(ui.display_count).to eq(1)
```

Comment:
display_count is the number of times display was called? Maybe there's a better name for for this method. In this case, you really just care that display was called, so you could change the spy to return a boolean from display_called? to achieve the same effect.